src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java

```diff
@@ -43,9 +35,9 @@ public class TimeoutCollector {
 
     /**
      * Create collector.
-     * 
-     * @param TTL       the cart entries' time to live in seconds
-     * @param taskRate  rate at which the collector checks the repo in milliseconds
+     *
+     * @param TTL      the cart entries' time to live in seconds
```

Comment:
It seems for params it still does alignments
I'm not sure if we want this

Reply:
Yes, I forgot to turn it off for Javadoc parameters.
But I think this actually helps reading the Javadoc as you then don't have to guess whether the next line is a new parameter or a continuation of the current parameter when skimming through.

Reply:
As decided (...by me in particular), Javadoc alignment is okay.